azure-pipelines.yml

```
@@ -82,6 +82,14 @@ jobs:
       - checkout: self
       - bash: ci/check-changelog.sh $(fork_sha)
 
+  - job: check_closing_quotes
+    pool:
+      name: 'linux-pool'
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: cd docs && scripts/check-closing-quotes.sh
```

This may not run reliably the same on CI and on your machine. Better to run it through dev-env.

```suggestion
      - bash: |
          set -euo pipefail
          ci/dev-env-install.sh
          eval $(dev-env/bin/dade-assist)
          cd docs && scripts/check-closing-quotes.sh
```

---

thanks, committing...